schema/pgsql/schema.sql

```diff
@@ -247,8 +261,8 @@ CREATE TABLE source (
     -- The hash is a PHP password_hash with PASSWORD_DEFAULT algorithm, defaulting to bcrypt. This check roughly ensures
     -- that listener_password_hash can only be populated with bcrypt hashes.
     -- https://icinga.com/docs/icinga-web/latest/doc/20-Advanced-Topics/#manual-user-creation-for-database-authentication-backend
-    CHECK (listener_password_hash IS NULL OR listener_password_hash LIKE '$2y$%'),
-    CHECK (type != 'icinga2' OR (icinga2_base_url IS NOT NULL AND icinga2_auth_user IS NOT NULL AND icinga2_auth_pass IS NOT NULL)),
+    CONSTRAINT ck_bcrypt_source_listener_password_hash CHECK (listener_password_hash IS NULL OR listener_password_hash LIKE '$2y$%'),
```

Comment:
This name seems to be an exception as it's not named `ck_${table}_...`  like the others.